docs/library/urandom.rst

```diff
@@ -0,0 +1,65 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator.
+
+Given below are the methods that this module provides.
+
+Function for integers
+---------------------
+ 
+  .. method:: urandom.getrandbits(n)
+
+    Returns an integer with 'n' random bits (1<=n<=32).
+    
+  .. method:: urandom.randint(a, b)
+    
+    Returns a random integer in the range [a, b].
+    
+  .. method:: urandom.randrange(stop)
+  
+    Returns an integer from the range [0, stop).
+
+Other Functions
+---------------
+  
+  .. method:: urandom.choice(sequence)
+   
+    Chooses multiple random items from a *sequence* (*list* , *set* or 
+    any data structure).
```

Comment:
chooses one item at random from the sequence and returns it

sets don't work (because have subscript capability), only objects that can be indexed with subscript notation will work (ie `o[idx]`)